packages/zero-tag-processor/src/utils/valueToLiteral.ts

```diff
@@ -0,0 +1,72 @@
+import type { types as t } from '@babel/core';
+import { type ExpressionValue, isSerializable } from '@linaria/utils';
+
+export default function valueToLiteral(value: unknown, ex?: ExpressionValue): t.Expression {
```

Comment:
```suggestion
export default function valueToLiteral(value: unknown, expression?: ExpressionValue): t.Expression {
```

Please use words for describing the args.